vpr/src/draw/draw.cpp

```diff
@@ -3997,6 +4066,577 @@ void net_max_fanout(GtkWidget* /*widget*/, gint /*response_id*/, gpointer /*data
     application.refresh_drawing();
 }
 
+//DEBUGGER SETTINGS
```

Comment:
This new code all looks weakly coupled to draw.cpp, which is good, but it means that it shouldn't be hard to move it into a new draw_debug.cpp file and that should be done.

Reply:
resolved